packages/orchestration/src/exos/chain-hub-admin.js

```diff
@@ -37,7 +37,7 @@ export const prepareChainHubAdmin = (zone, chainHub) => {
        * @param {CosmosChainInfo} chainInfo
        * @param {IBCConnectionInfo} connectionInfo
        */
-      async initChain(chainName, chainInfo, connectionInfo) {
+      async registerChain(chainName, chainInfo, connectionInfo) {
```

Comment:
better name. I forget why we had `init` before. Maybe thinking it would be only allowed once? but clearly what it's doing is registering